gitlab/visualgithubprdiscussion.py

```diff
@@ -18,7 +18,7 @@ def toMessage(url):
     index=message.find(BEGIN_RGX)
     if index>-1:
         for line in message[index:].split('\n')[1:]:
```

Comment:
Probably clearer to use `splitlines()` here and do away with the strip below?

Reply:
Indeed better, is implemented now.

Locally you can see the difference now with:
```python3 gitlab/visualgithubprdiscussion.py public 910```
Where public is in the regex and is therefor a wanted change
```python3 gitlab/visualgithubprdiscussion.py anotherendpoint 910```
which is not in the regex and therefor would be a failing change